_delphi_utils_python/tests/test_geomap.py

```diff
@@ -411,6 +420,23 @@ def test_add_geocode(self):
             )
         )
 
+        # hrr -> nation
+        with pytest.raises(ValueError):    
+            new_data = gmpr.replace_geocode(self.zip_data, "zip", "hrr")
```

Comment:
I'm pretty sure if the first line valueerrors but the second one doesnt, it still passes, so may want to split this into two `with pytest raises....` statements

Reply:
Wouldn't that have been caught in the zip -> hrr test some lines before that though?

Reply:
Maybe im understanding this test wrong. I read it as testing that both 

```
            new_data = gmpr.replace_geocode(self.zip_data, "zip", "hrr")
```
and
```
            new_data2 = gmpr.replace_geocode(new_data, "hrr", "nation")
```
raise valueerrors. Is that right?

Reply:
It's testing the second one, since we should not be mapping hrr -> nation (it's an incomplete mapping, so it's unsupported).

Reply:
Ohhh got it, didn't see the second line calls new_data. Thanks.